### DISCUSSION TITLE
alist新版本WebDav挂载百度网盘，无法上传文件，提示“你需要权限来执行此操作”。官方是否不打算解决了？

### DISCUSSION THREAD

**Discussion by xinanzai** [AlistGo/alist#4656] [General]
看到之前的讨论，https://github.com/alist-org/alist/discussions/1639

这个问题下并没有明确的解决方案，官方是否不打算解决了，如果百度网盘无法像之前2.x版本一样挂载到本地正常使用，只能下载不能上传，那使用alist的意义在哪里，为什么不直接使用百度云客户端呢。。。。alist的更新如果脱离了实用价值，更新到底还有什么意义

**Comment by xhofe** [comment]
解决方案：用`rclone`而不是`RaiDrive`挂载。
